index.html

```diff
@@ -2362,6 +2367,94 @@ <h3>
           can get their games and apps rated with IARC</a>.
         </p>
       </section>
+      <section>
+        <h3>
+          <code>shortcuts</code> member
+        </h3>
+        <p>
+          The <dfn>shortcuts</dfn> member is an <a>array</a> of
+          <a>ShortcutItem</a>s that provide access to key tasks within a web
+          application. For example, they can be used to link directly to a
+          user’s timeline within a social media application or their recent
+          orders in an ecommerce context.
```

Comment:
e-commerce?

Reply:
Fixing now. Is there a style guide and favored dictionary for specs?

Reply:
I don't believe so, but US English is the norm